tests/behaviour/expectations/behaviour.ts

```diff
@@ -986,6 +986,20 @@ export const expectations = flatten(
           Expect.Simple('test', [], []),
           Expect.Simple('returnTest', [], ['12', '0']),
         ]),
+        new Dir('modifiers', [
```

Comment:
Test expectations should be laid out in alphabetical order at each level (at least for all the Files and Dirs, doesn't really matter for the Expects). With this many tests, they get lost otherwise